models/models.go

```diff
@@ -19,7 +19,10 @@ type StepListItemModel map[string]stepmanModels.StepModel
 
 // PipelineModel ...
 type PipelineModel struct {
-	Stages []StageListItemModel `json:"stages,omitempty" yaml:"stages,omitempty"`
+	Stages      []StageListItemModel `json:"stages,omitempty" yaml:"stages,omitempty"`
+	Title       string               `json:"title,omitempty" yaml:"title,omitempty"`
```

Comment:
Pls move `Title`, `Summary` and `Description` to the top of the struct fields, to follow the convention.

Reply:
done